src/ekat/logging/ekat_logger.hpp

```diff
@@ -50,36 +50,57 @@ namespace Log = spdlog;
   of its functionality.
 */
 template <typename LogFilePolicy = LogNoFile,    // see ekat_log_file.hpp
-          typename LogMpiPolicy = LogIgnoreRank> // see ekat_log_mpi.hpp
+          typename LogMpiPolicy = LogOnlyRank0> // see ekat_log_mpi.hpp
 class Logger : public spdlog::logger {
   public:
-    Logger() = delete;
 
-    Logger(const std::string& log_name, const std::string& level_str="debug") :
-     spdlog::logger(LogMpiPolicy::name_with_rank(log_name))
+  // primary constructor
+  Logger(const std::string& log_name, const Log::level::level_enum lev=Log::level::debug,
```

Comment:
I personally would not default the comm argument. Someone will use the version with defaulted comm (knowing what they are doing), then someone will copy that piece of code (not knowing what they are doing), and all of a sudden we'll have weird logs. If one is ok with a default comm (which wraps MPI_COMM_SELF, which is likely not desired), then they can do so explicitly:
```
Logger<FP,MP> logger("my_log","debug",ekat::Comm());
```